src/main_app/db/create_helper.py

```
@@ -51,8 +51,15 @@ def create_views(_db: SQLAlchemy) -> None:
                 with conn.begin():
                     conn.execute(text(create_sql))
-            except Exception:
-                logger.error("Failed to create view %s", table.name)
+            except OperationalError as exc:
+                code = getattr(exc, "code", None)
+                if code == 1050:
+                    logger.warning("View %s already exists, skipping", table.name)
+                else:
+                    logger.exception("Failed to create view %s", table.name)
+
+            except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)